### PR TITLE
Update CI to Humble

### PR DESCRIPTION
**Pull request by luca-della-vedova**
### Fixed bug

Update CI to newest setup-ros action / Humble and Ubuntu Jammy, as done in https://github.com/open-rmf/rmf_simulation/pull/77

### CODE REVIEW

**Comment by Yadunund** [comment]
Could you also copy over the style CI? And update the readme to display both badges

**Comment by Yadunund** [comment]
And wondering if this should target the `feature/rmf_obstacle` branch?

**Comment by luca-della-vedova** [comment]
All done! Retargeting the rmf_obstacle branch, had to add a few code style changes (result of `ament_uncrustify --reformat`) to make CI green.
I had a bit of trouble with `${rmf_utils_DIR}` not being populated for a few packages so I added an explicit `find_package(rmf_utils REQUIRED)` for test builds, as well as a `test_dependency` in package.xml to fix them.

CI is now green, merging this in the `rmf_obstacle` branch should also make its CI green and unblock the PR

File: rmf_visualization_navgraphs/src/NavGraphVisualizer.cpp [hunk 1]
```diff
@@ -203,7 +203,8 @@ auto NavGraphVisualizer::FleetNavGraph::update_lane_states(
   std::optional<std::string> map_name_filter) -> std::vector<Marker::ConstSharedPtr>
 {
   if (!traffic_graph.has_value())
-    return {};
+    return {}
```
Comment (Yadunund) [inline rmf_visualization_navgraphs/src/NavGraphVisualizer.cpp:206]: This diff looks a bit weird. I think if we wrap that if inside braces, the semicolon will not move into a newline.

**Review by Yadunund — COMMENTED** [review]
Thanks for tackling this issue. Looks good! Just have one very tiny nitpick on what uncrustify did

**Review by Yadunund — APPROVED** [review]

**Pull request merged**